cmd/spire-server/cli/bundle/set.go

```diff
@@ -0,0 +1,109 @@
+package bundle
+
+import (
+	"context"
+	"errors"
+	"flag"
+	"fmt"
+	"io"
+	"io/ioutil"
+	"os"
+
+	"github.com/mitchellh/cli"
+	"github.com/spiffe/spire/pkg/common/idutil"
+	"github.com/spiffe/spire/pkg/common/pemutil"
+	"github.com/spiffe/spire/proto/api/registration"
+)
+
+// NewSetCommand creates a new "set" subcommand for "bundle" command.
+func NewSetCommand() cli.Command {
+	return newSetCommand(defaultEnv, newClients)
+}
+
+func newSetCommand(env *env, clientsMaker clientsMaker) cli.Command {
+	return adaptCommand(env, clientsMaker, new(setCommand))
+}
+
+type setCommand struct {
+	// SPIFFE ID of the trust bundle
+	id string
+
+	// Path to the bundle on disk (optional). If empty, reads from stdin.
+	path string
+}
+
+func (c *setCommand) name() string {
+	return "bundle set"
+}
+
+func (c *setCommand) synopsis() string {
+	return "Sets bundle data"
```

Comment:
can we say that this does a create or update ?

Reply:
Sure, that sounds good to me.